src/main/java/org/ossgang/commons/observable/Observable.java

```diff
@@ -22,12 +22,14 @@
 
 package org.ossgang.commons.observable;
 
+import static org.ossgang.commons.observable.DerivedObservableValue.derive;
```

Comment:
Duplicated import? Some GitHub problem?

Reply:
Yes indeed. my fault! Did not update/rebase correctly.